.circleci/config.yml

```diff
@@ -492,6 +492,8 @@ commands:
           environment:
             # Use the settings from the "ci" profile in nextest configuration.
             NEXTEST_PROFILE: ci
+            # Temporary disable lib backtrace since it crashing on MacOS, if xcode < 15.1.0
+            RUST_LIB_BACKTRACE: 0
```

Comment:
I did this for all platforms since it doesn't influence our tests.
Also these backtraces are not printed in logs anyway.
But keeping behavior unified for all platforms makes the test more predictable.

Reply:
so we would never have a backtrace for a failing test in CI?